packages/@sanity/client/src/config.js

```diff
@@ -10,29 +11,31 @@ const defaultConfig = {
   isPromiseAPI: true
 }
 
-const cdnWarning = [
+const LOCALHOSTS = ['localhost', '127.0.0.1', '0.0.0.0']
+const isLocal = host => LOCALHOSTS.indexOf(host) !== -1
+
+// eslint-disable-next-line no-console
+const printWarning = message => once(() => console.warn(message.join(' ')))
```

Comment:
Shouldn't this be `getWarningPrinter` or something? The `printWarning` calls below confused me quite a bit since there were no conditional around them